chat-model/src/main/resources/canon/chat.json

```diff
@@ -1261,6 +1261,29 @@
           }
         }
       },
+      "ReactionDetailsPayload": {
+        "type": "object",
+        "extends": "#/components/schemas/MaestroPayload",
+        "properties": {
+          "action": {
+            "type": "string"
+          },
+          "reactionName": {
+            "type": "string"
+          },
+          "messageId": {
+            "type": "string"
+          },
+          "userId": {
+            "type": "integer",
+            "format": "int64"
```

Comment:
Again, I would give this a more descriptive name, what does this mean and (assuming this is a symphony user ID) the type should be

"userId": {
            "$ref": "https://models.oss.symphony.com/core.json#/components/schemas/PodAndUserId"
          }

Reply:
I updated the type.